Session/Home/HomeVC.swift

```diff
@@ -256,9 +305,7 @@ final class HomeVC : BaseVC, UITableViewDataSource, UITableViewDelegate, NewConv
     }
     
     @objc private func handleSeedViewedNotification(_ notification: Notification) {
-        tableViewTopConstraint.isActive = false
-        tableViewTopConstraint = tableView.pin(.top, to: .top, of: view, withInset: Values.smallSpacing)
-        seedReminderView.removeFromSuperview()
+        self.tableView.reloadSections(IndexSet(integer: 0), with: .none) // TODO: Just reload header if possible
```

Comment:
`TODO` left in the code

Reply:
Yeah, should be a `FIXME`, but as we change the UI, this is not in use anymore. :P